master/master.go

```diff
@@ -7,9 +7,9 @@ import (
 )
 
 type Master struct {
-	workers     []Worker
-	idleWorkers chan Worker
-	linksCh     chan []string
+	workerClients     []WorkerClient
+	idleWorkerClients chan WorkerClient
```

Comment:
```suggestion
	workers     []WorkerClient
	idleWorkers chan WorkerClient
```
Thank you making the changes. I feel calling them workers is more natural in this code. You may disagree though.